libs/positions/src/lib/positions-table.tsx

```diff
@@ -64,33 +66,6 @@ export const MarketNameCell = ({ valueFormatted }: MarketNameCellProps) => {
   return (valueFormatted && valueFormatted[0]) || undefined;
 };
 
-export interface PriceCellProps {
```

Comment:
nothing really changed here. just moved some stuff to ui-toolkit

Reply:
Hm if there's a circular dependency here then something is off elsewhere as the other cell components are working from there fine. Lets fix later though!